app/src/main/java/org/simple/clinic/home/overdue/OverdueScreen.kt

```diff
@@ -172,18 +147,22 @@ class OverdueScreen : BaseScreen<
     country.isoCountryCode == Country.INDIA
   }
 
+  private val uiModelsState = mutableStateOf<List<OverdueUiModel>>(emptyList())
```

Comment:
We can make this a var, and use `by` operator here directly.

Reply:
Done